.github/workflows/_charm-linting.yaml

```diff
@@ -23,4 +23,11 @@ jobs:
       - name: Install dependencies
         run: python3 -m pip install tox
       - name: Run linters
-        run: cd ${{ inputs.charm-path }} && tox -vve lint
+        run: |
+          cd ${{ inputs.charm-path }}
+          if [ -f tox.ini ]; then
+            tox -vve lint
+          else
```

Comment:
shouldn't we check if there's a makefile? or, perhaps a wild idea, shouldn't we revert this whole flow and do this:

- always install uv
- if there is no makefile, then:
  - create a makefile which contains:
  - make lint --> tox -e lint
  - make unit --> tox -e scenario unit
  - ...